packages/fiber/src/native/Canvas.tsx

```diff
@@ -59,19 +43,18 @@ class ErrorBoundary extends React.Component<{ set: React.Dispatch<any> }, { erro
 }
 
 export const Canvas = /*#__PURE__*/ React.forwardRef<View, Props>(
-  ({ children, fallback, style, events, ...props }, forwardedRef) => {
+  ({ children, fallback, style, events, onPointerMissed: _onPointerMissed, ...props }, forwardedRef) => {
     // Create a known catalogue of Threejs-native elements
     // This will include the entire THREE namespace by default, users can extend
     // their own elements by using the createRoot API instead
     React.useMemo(() => extend(THREE), [])
-    const onPointerMissed = useMemoizedFn(props.onPointerMissed)
+    const onPointerMissed = useMemoizedFn(_onPointerMissed)
 
     const [{ width, height }, setSize] = React.useState({ width: 0, height: 0 })
     const [canvas, setCanvas] = React.useState<HTMLCanvasElement | null>(null)
     const [bind, setBind] = React.useState<any>()
 
-    const canvasProps = pick<Props>({ ...props, onPointerMissed }, CANVAS_PROPS)
-    const viewProps = omit<Props>({ ...props, onPointerMissed }, CANVAS_PROPS)
+    const { canvasProps, wrapperProps: viewProps } = extractCanvasProps('native', { ...props, onPointerMissed })
```

Comment:
I'd prefer to destructure these props above if this is confined to the canvas.

Reply:
Sorry, I'm afraid I don't catch your point, do you mean the `const {canvasProp, wrapperProps: viewProps}...` or the `{...props, onPointerMissed}` ?